CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 0.26.0 / 2023-04-03
+
+- [FEATURE] Add __experimentalEChartsOptionsOverride prop to line chart component (#1023)
+- [FEATURE] Stack option support in time series panel (#997)
+- [ENHANCEMENT] always show info icon when panel has a description (#1059)
+- [ENHANCEMENT] PrometheusLabelValuesVariable plugin: Improve the migration logic to preserve the eventual matcher
+- [ENHANCEMENT] Ensure all bad request error has the same format (#1043)
+- [ENHANCEMENT] small improvements to the Migrate feature - backend: update plugins mig.cuepart files so that migration don't fail the on empty panels (no query) - Migrate screen: more meaningful errors on wrong inputs - Migrate screen: add missing margin to container - Migrate screen: adjust warning message
```

Comment:
is there any relevant pull request that should be mentioned?

Reply:
Lemme double-check. They probably got lost in a squash or something like that.